.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-- 6.2.0
-- 10
+- 8.10.0
+- 14
```

Comment:
And maybe use just `node` here for the very latest (stable) release?`

See https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions